conf/gravitino-iceberg-rest-server.conf.template

```diff
@@ -41,7 +41,23 @@ gravitino.iceberg-rest.requestHeaderSize = 131072
 gravitino.iceberg-rest.responseHeaderSize = 131072
 
 # THE CONFIGURATION FOR Iceberg catalog backend
-# The Iceberg catalog backend, it's recommended to change to hive or jdbc
+# The Iceberg catalog backend, it's recommanded to change to hive or jdbc 
```

Comment:
"recommended", also remove the trailing whitespace. This line is not necessary to change.

Reply:
done